wiki/markdown/usage.md

```diff
@@ -0,0 +1,7 @@
+## Usage
+
+To get information about configuration and usage of **REMReM Generate CLI** see [here](generate-cli.md).
+
+To get information about configuration and usage of **REMReM Generate Service** see [here](generate-service.md).
+
+Examples used here are for the next generation of Eiffel events called semantics, which is open source. Please follow [eiffel syntax](https://github.com/eiffel-community/eiffel/tree/master/eiffel-syntax-and-usage) and [eiffel vocabulary](https://github.com/eiffel-community/eiffel/tree/master/eiffel-vocabulary).
```

Comment:
Rephrase this sentence to remove any references to different Eiffel generations. How about: "Example uses of Eiffel events can be found in the protocol repository" or something similar?